client/src/components/Headers/Navbar.tsx

```diff
@@ -0,0 +1,99 @@
+          aria-expanded={isMenuOpen ? "true" : "false"}
+        >
+          <span className="sr-only">Open main menu</span>
+          <svg className="w-5 h-5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 17 14">
+            <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M1 1h15M1 7h15M1 13h15" />
+          </svg>
+        </button>
+        <div className={`w-full md:block md:w-auto ${isMenuOpen ? '' : 'hidden'}`} id="navbar-default">
+          
+          <ul className={`font-medium flex items-center flex-col p-4 md:p-0 mt-4 border border-gray-100 rounded-lg ${theme === 'light' ? 'bg-gray-50 md:bg-transparent' : 'bg-[#12141d] md:bg-transparent'} md:flex-row md:space-x-8 rtl:space-x-reverse md:mt-0 md:border-0`}>
+          {
+            landingPage &&
+            <>
+            <li>
+              <ScrollLink to='/' smooth duration={500}>
+                <p className="block py-2 px-3 text-white bg-blue-700 rounded md:bg-transparent md:text-blue-700 md:p-0" aria-current="page">Home</p>
+              </ScrollLink>
+            </li>
+            <li>
+              <ScrollLink to='about' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>About</p>
+              </ScrollLink>
+            </li>
+            {/* <li>
+              <ScrollLink to='' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Services</p>
+                </ScrollLink>
+              </li> */}
+            <li>
+              <ScrollLink to='contact' smooth duration={500}>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Contact</p>
+              </ScrollLink>
+            </li>
+            <li>
+              <Link href='/auth/register'>
+                <p className={`block py-2 px-3 ${theme === 'light' ? 'text-gray-900' : 'text-white'} rounded hover:bg-gray-100 md:hover:bg-transparent md:border-0 md:hover:text-blue-700 md:p-0`}>Register</p>
+              </Link>
+            </li>
+            <li>
+              <Link data-testid="Login-link" href='/auth/login'>
+                <button className='border-blue-700 border hover:bg-blue-700 hover:text-white hover:ring rounded px-3 py-1 text-blue-700 bg-transparent'>
+                  Login
+                </button>
+              </Link>
+            </li>
+              </>}
+            <li>
+              <button className='p-1 md:mt-0 mt-2' onClick={toggleTheme}>
+                {
+                  theme === 'light' && 
```

Comment:
use ternary operator like this everywhere: 
`theme === 'light' ? <svg> : null` 

Reason it's a good practice to use this.